### PR TITLE
Added files

### PR DESCRIPTION
**Pull request by Tzurrr**

### CODE REVIEW

File: elogger.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import json
+import logging
+import ecs_logging
+import requests
+import uuid
+import os
+
+def write(event_string):
```
Comment (nadav5200) [inline elogger.py:8]: not meaningful enough function name

File: elogger.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    logger = logging.getLogger("app")
+    logger.setLevel(logging.DEBUG)
+    handler = logging.FileHandler('/home/tzur/elvis.json')
+    handler.setFormatter(ecs_logging.StdlibFormatter())
+    logger.addHandler(handler)
+    json_UUID = uuid.uuid4()
+    keywords_arr = ["arrivedtoserver", "sent", "didntsent", "wrote"]
+    
+    # TODO: add value check
```
Comment (nadav5200) [inline elogger.py:17]: remove comments

File: elogger.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+    logger.info(event_string, extra={"http.request.method": "get", "UUID": json_UUID})
+    log_content = open("/home/tzur/elvis.json", "r")
+    log_content_json = json.load(log_content)
+    log_content.close()
+
+    os.remove("/home/tzur/elvis.json")
+
+    doc_UUID = uuid.uuid4()
+    resp = requests.put(url=f"http://13.81.211.207:9200/{event_string}/_doc/{doc_UUID}", json=log_content_json,
```
Comment (nadav5200) [inline elogger.py:27]: urls should be in a config file, and use a different method instead of put

File: elogger.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    keywords_arr = ["arrivedtoserver", "sent", "didntsent", "wrote"]
+    
+    # TODO: add value check
+
+    logger.info(event_string, extra={"http.request.method": "get", "UUID": json_UUID})
+    log_content = open("/home/tzur/elvis.json", "r")
+    log_content_json = json.load(log_content)
+    log_content.close()
+
+    os.remove("/home/tzur/elvis.json")
```
Comment (nadav5200) [inline elogger.py:24]: move to a different function and wrap with try and except

File: encryptor.py [hunk 5]
```diff
@@ -0,0 +1,45 @@
+import base64
+import hashlib
+from Crypto import Random
+from Crypto.Cipher import AES
+
+
+def encrypt(filename):
+    # hash
```
Comment (nadav5200) [inline encryptor.py:8]: remove the comments

File: encryptor.py [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+import hashlib
+from Crypto import Random
+from Crypto.Cipher import AES
+
+
+def encrypt(filename):
+    # hash
+ #   print(filename)
+    with open(filename, "rb") as file:
+        file.seek(0)
```
Comment (nadav5200) [inline encryptor.py:11]: what is the reason for the seek here?

File: encryptor.py [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+        content = file.read()
+
+    a = hashlib.sha512(str(content).encode())
```
Comment (nadav5200) [inline encryptor.py:14]: not meaningful variable name

File: encryptor.py [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+    a = a.hexdigest()
+ #   print(a)
+
+    # iv for obj props
+    iv = Random.new().read(AES.block_size)
+#    print(iv)
+
+    # getting the key which is also for generating the obj props
+    with open("/home/tzur/server-tools1/keys/tornado.key", "rb") as file:#"/home/tzur/keys/tornado.key", "rb") as file:
+        key = file.read() + b"    "  # [:24]
```
Comment (nadav5200) [inline encryptor.py:24]: what is the reason for the b"    "

File: encryptor.py [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+
+    # creating the obj
+    obj = AES.new(key, AES.MODE_CFB, iv)
+
+    # the hash encrypted with the object
+    final_obj = base64.b64encode(obj.encrypt(a.encode()))
+
+    with open(filename, "ab") as file:
+        print((iv, final_obj))
+        file.write(iv)
```
Comment (nadav5200) [inline encryptor.py:34]: no need for the write function twice

File: encryptor.py [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+        file.write(final_obj)
+
+
+
+def to_bytes(s):
```
Comment (nadav5200) [inline encryptor.py:39]: you don't use this function anywhere

File: find_dot.py [hunk 11]
```diff
@@ -0,0 +1,7 @@
+def find(filename):
```
Comment (nadav5200) [inline find_dot.py:1]: there is a bulit-in function that do the same as here

File: keys/aaa.key [hunk 12]
```diff
@@ -0,0 +1 @@
+qqqqqqqqqqqqewrwqere
```
Comment (nadav5200) [inline keys/aaa.key:1]: remove this key

File: todo [hunk 13]
```diff
@@ -0,0 +1,2 @@
+# add a config file that includes paths, etc.
```
Comment (nadav5200) [inline todo:1]: remove the todo

File: final_server.py [hunk 14]
```diff
@@ -0,0 +1,28 @@
...
+import encryptor
+import elogger
+
+app = FastAPI()
+
+@app.post("/")
+async def upload(files: List[UploadFile] = File(...)):
+    contents = []
+    for file in files:
+        await file.seek(0)
```
Comment (nadav5200) [inline final_server.py:14]: no need to use seek here

File: final_server.py [hunk 15]
```diff
@@ -0,0 +1,28 @@
...
+import find_dot
+import encryptor
+import elogger
+
+app = FastAPI()
+
+@app.post("/")
+async def upload(files: List[UploadFile] = File(...)):
+    contents = []
+    for file in files:
```
Comment (nadav5200) [inline final_server.py:13]: move all this for logic to a different function

File: final_server.py [hunk 16]
```diff
@@ -0,0 +1,28 @@
...
+        await file.seek(0)
+        contents.append(await file.read())
+        await file.close()
+        dot = find_dot.find(file.filename)
```
Comment (nadav5200) [inline final_server.py:17]: not meaningful enough variable name

File: final_server.py [hunk 17]
```diff
@@ -0,0 +1,28 @@
...
+    with open (f"/home/tzur/all-the-photos/{file.filename[:dot-2]}.jpg", "wb") as file:
+        file.writelines(contents)
+        filename = file.name
+    elogger.write("wrote")
+    encryptor.encrypt(filename)
+    return {"message": f"Successfuly uploaded {[file.filename for file in files]}"}
+
+if __name__ == '__main__':
+    uvicorn.run(app, host='0.0.0.0', port=8000)
+    intro = "/home/tzur/getters/"
```
Comment (nadav5200) [inline final_server.py:27]: irrelevant variable

File: final_server.py [hunk 18]
```diff
@@ -0,0 +1,28 @@
...
+        dot = find_dot.find(file.filename)
+    with open (f"/home/tzur/all-the-photos/{file.filename[:dot-2]}.jpg", "wb") as file:
+        file.writelines(contents)
+        filename = file.name
+    elogger.write("wrote")
+    encryptor.encrypt(filename)
+    return {"message": f"Successfuly uploaded {[file.filename for file in files]}"}
+
+if __name__ == '__main__':
+    uvicorn.run(app, host='0.0.0.0', port=8000)
```
Comment (nadav5200) [inline final_server.py:26]: move this to a config file

File: final_server.py [hunk 19]
```diff
@@ -0,0 +1,28 @@
...
+
+@app.post("/")
+async def upload(files: List[UploadFile] = File(...)):
+    contents = []
+    for file in files:
+        await file.seek(0)
+        contents.append(await file.read())
+        await file.close()
+        dot = find_dot.find(file.filename)
+    with open (f"/home/tzur/all-the-photos/{file.filename[:dot-2]}.jpg", "wb") as file:
```
Comment (nadav5200) [inline final_server.py:18]: you should use a relative path and not a specific path

File: final_server.py [hunk 20]
```diff
@@ -0,0 +1,28 @@
...
+from fastapi import File, UploadFile, FastAPI
+from typing import List
+import find_dot
+import encryptor
+import elogger
+
+app = FastAPI()
+
+@app.post("/")
+async def upload(files: List[UploadFile] = File(...)):
```
Comment (nadav5200) [inline final_server.py:11]: find a better name for the function

File: elogger.py (hunk 1 above)
Comment (Tzurrr) [reply elogger.py:8]: V

File: elogger.py (hunk 2 above)
Comment (Tzurrr) [reply elogger.py:17]: V

File: elogger.py (hunk 4 above)
Comment (Tzurrr) [reply elogger.py]: V

File: encryptor.py (hunk 5 above)
Comment (Tzurrr) [reply encryptor.py:8]: V

File: encryptor.py (hunk 6 above)
Comment (Tzurrr) [reply encryptor.py:11]: V

File: encryptor.py (hunk 7 above)
Comment (Tzurrr) [reply encryptor.py:14]: V

File: keys/aaa.key (hunk 12 above)
Comment (Tzurrr) [reply keys/aaa.key:1]: V

File: todo (hunk 13 above)
Comment (Tzurrr) [reply todo]: V

File: final_server.py (hunk 14 above)
Comment (Tzurrr) [reply final_server.py:14]: V

File: final_server.py (hunk 15 above)
Comment (Tzurrr) [reply final_server.py:13]: V

File: final_server.py (hunk 16 above)
Comment (Tzurrr) [reply final_server.py:17]: V

File: final_server.py (hunk 17 above)
Comment (Tzurrr) [reply final_server.py:27]: V

File: final_server.py (hunk 18 above)
Comment (Tzurrr) [reply final_server.py:26]: V

File: find_dot.py (hunk 11 above)
Comment (Tzurrr) [reply find_dot.py:1]: V

File: final_server.py (hunk 20 above)
Comment (Tzurrr) [reply final_server.py:11]: V

**Pull request closed without merging**